LoopKit/GlucoseKit/GlucoseStore.swift

```diff
@@ -636,3 +649,101 @@ extension GlucoseStore {
         }
     }
 }
+
+extension NSManagedObjectContext {
+    fileprivate func cachedGlucoseObjectsWithUUID(_ uuid: UUID, fetchLimit: Int? = nil) -> [CachedGlucoseObject] {
```

Comment:
[nit] minor suggestion to follow the pattern of having functions start with verbs, like `purgeCachedGlucoseObjects` above, this maybe could be `fetchCachedGlucoseObjectsWithUUID` and the one below `fetchCachedGlucoseObjectsWithSyncIdentifier`.

Reply:
This was simply copy/pasted from a separate file that contained just this extension into this file so that all code that modified this Core Data store was within and private to the file. (It makes is easier to reason about making changes to the store if all code is within one file as opposed to spread out within extension across multiple files.)

Technically this code doesn't even need to be in an extension to NSManagedObjectContext and should be in the GlucoseStore class itself, but making that change was not required nor suggested by my changes.

Reply:
Gotcha.  Again, this and the comment below was just a [nit].  So no obligation whatsoever.